pages/models/regulations/governors/GovSteamEU/index.md

```diff
@@ -0,0 +1,422 @@
+the unit. Also for some stability phenomena (e.g. resonance stability)
+this model is not sufficient and EMT models or other approaches may be
+necessary .
+
+The model operates only under the assumption that resistive and
+mechanical losses are negligible, and in steady state condition
+$$P_\mathrm{e} = P_\mathrm{m}$$. This assumes that mechanical losses do
+not have any major effect on dynamic stability [1].
+
+## Model description
+
+### Turbine model
+
+The turbine model is the mechanical part of the model with boiler, high
+pressure (HP) turbine and low pressure (LP) turbine [2]. Its output is
+the mechanical power $$P_\mathrm{m}$$. A schematic representation of a
+steam turbine with HP and LP part is shown in
+<a href="#fig-Steam_turbine_configuration"
+class="quarto-xref">Figure 2</a>. The HP stage’s single-triangle symbol
+and the LP stage’s double-triangle symbol represent the fact that in the
+HP stage steam expands in only one direction whereas in the LP stage it
+expands in two directions [3].
+
+#### Boiler
+
+The boiler is responsible for converting water into high-pressure,
+high-temperature steam. In normal operation $$P_\mathrm{set}$$ is used as
+the boiler set-point $$P_\mathrm{set\,boiler}$$, i.e. the heat supplied to
+the boiler. In over speed situation, the emergency speed control signal
+is used instead, see
+<a href="#sec-emergency" class="quarto-xref">Section 3.2.3</a>.
+
+The boiler output, i.e. the amount of steam supplied from the boiler to
+the turbine, is manipulated by the control valves [2].
+
+#### High-Pressure (HP) and low pressure (LP) turbine stages
+
+The HP and LP stages are modeled by a first order elements and gains
+$$T_\mathrm{hp}$$, $$K_\mathrm{hp}$$ and $$T_\mathrm{lp}$$, $$K_\mathrm{lp}$$.
+
+Steam power from the boiler enters the HP stage and is partly converted
+into mechanical power. The steam output from the HP stage then enters
+the reheater. Because the HP stage’s dynamic response is relevant here,
+the output of the $$T_\mathrm{hp}$$ lag element is used as input, but
+without the gain $$K_\mathrm{hp}$$.
+
+Before entering the LP stage, the output steam from the HP stage must be
+reheated to prevent any water droplets from entering and to increase
+efficiency. The reheater is also modeled by a first order element with
+time constant $$T_\mathrm{rh}$$.
+
+From there the reheated steam passes the intercept valve and enters the
+LP stage. Here, the first order element directly contains the gain
+$$K_\mathrm{lp}$$, because only the generated power is of interest (the
+dynamic behaviour of the steam flow without gain is not necessary
+because there is no other following turbine stage) [2].
+
+The mechanical power output $$P_\mathrm{m}$$ is the sum of the parts
+generated by the HP and LP turbine stages. To accurately represent this
+distribution, the gains for the HP and LP turbine stages must satisfy
+the following condition:
+
+$$K_\mathrm{hp} + K_\mathrm{lp}  = 1$$
+
+### Control paths
```

Comment:
is there a way we can see the control paths on the overall complete diagram or zooming with different colors or frames?

Reply:
I will look into it

Reply:
I added the controlpaths to the drawing -- please have a look what you think!